### ISSUE TITLE
[label bot] install Tekton on issue-label-bot cluster

### ISSUE THREAD

**Issue by jlewi** [kubeflow/code-intelligence#153]
Tracking issue for installing Tekton on issue-label-bot cluster

Per: https://github.com/tektoncd/pipeline/issues/2667

Tekton requires 1.15 our cluster is currently 1.14 so we need to upgrade.

Will switch to regular channel:
https://cloud.google.com/kubernetes-engine/docs/release-notes-regular

**Comment by jlewi** [comment]
```
gcloud --project=issue-label-bot-dev beta container clusters update issue-label-bot \
    --release-channel regular
ERROR: (gcloud.beta.container.clusters.update) INVALID_ARGUMENT: Master version 1.14.10-gke.36 cannot be upgraded to releaseChannel REGULAR default version 1.16.9-gke.6: cannot upgrade the master more than a minor version at a time.

```

**Comment by jlewi** [comment]
Lets try upgrading to `1.15` first

**Comment by jlewi** [comment]
Upgrade to 1.15.12-gke.6 worked
Now lets upgrade to 1.16.9-gke.6 to bring it in sync with regular channel.

**Comment by jlewi** [comment]
Upgrade to 1.16.9 worked.

**Comment by jlewi** [comment]
Enrolled in regular channel

```
gcloud --project=issue-label-bot-dev beta container clusters update issue-label-bot \
    --release-channel regular
Updating issue-label-bot...done.                                                                                                                                                                                   
Updated [https://container.googleapis.com/v1beta1/projects/issue-label-bot-dev/zones/us-east1-d/clusters/issue-label-bot].
To inspect the contents of your cluster, go to: https://console.cloud.google.com/kubernetes/workload_/gcloud/us-east1-d/issue-label-bot?project=issue-label-bot-dev

```

**Comment by jlewi** [comment]
Fixed by #154
